packages/open-payments/src/client/token.test.ts

```diff
@@ -27,11 +27,15 @@ describe('token', (): void => {
       jest.spyOn(openApi, 'createResponseValidator')
       createTokenRoutes({ axiosInstance, openApi, logger })
 
-      expect(openApi.createResponseValidator).toHaveBeenCalledTimes(1)
+      expect(openApi.createResponseValidator).toHaveBeenCalledTimes(2)
       expect(openApi.createResponseValidator).toHaveBeenCalledWith({
         path: '/token/{id}',
         method: HttpMethod.POST
       })
+      expect(openApi.createResponseValidator).toHaveBeenCalledWith({
+        path: '/token/{id}',
+        method: HttpMethod.DELETE
+      })
```

Comment:
@domin191013 let's try adding a test similar to: https://github.com/interledger/rafiki/pull/851/commits/c1be7b2e2edd4a913f94bc6623ccf2f0e7cfb91f

This is so that we can test that the correct validator gets called for the route